cmd/core/main.go

```diff
@@ -205,12 +206,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	plainValidator, err := validators.NewPlainValidator()
+	if err != nil {
+		setupLog.Error(err, "unable to setup bundle validator")
+		os.Exit(1)
+	}
```

Comment:
Do we plan to use a different validator with the plain provisioner? If not, I'd suggest hiding this as an implementation detail of the plain bundle reconciler rather than making it a configurable thing on the struct.

Reply:
I wasn't sure, but I imagined but I figured the registry provisioner (or any other "conversion" provisioners) would use this as well.

I may have over engineered, but given the recent history of operator-lib validations I am imagining that the reconcilers may need to use different versions of the same validators, or that reconcilers might share validators.

Reply:
> I imagined the registry provisioner (or any other "conversion" provisioners) would use this as well.

I could see that as a sort of post-conversion step perhaps, but if our registry+v1 to plain+v0 conversion resulted in a failure in the plain bundle validation, that'd be a bug on our side, not really something that would be useful/actionable for a user.

Perhaps it's worth structuring it such that the registry+v1 to plain+v0 conversion tests can call it as a sanity check?